src/SFA.DAS.Commitments.Database/StoredProcedures/GetActiveApprenticeshipsByULNs.sql

```diff
@@ -18,5 +20,6 @@ INNER JOIN [dbo].[Apprenticeship] a ON u.ULN = a.ULN
 INNER JOIN [dbo].[Commitment] c ON c.Id = a.CommitmentId
 WHERE
 a.AgreementStatus = 3 -- Both parties agreed - "right of the line"
-AND a.PaymentStatus IN (1,2,3) --Active, Paused or Cancelled
+AND a.PaymentStatus <= 3 --PendingApproval, Active, Paused or Cancelled
```

Comment:
We might actually want to consider including Completed status also. It's not a value currently used, but if it were to be introduced I imagine we'd still want completed apprenticeships to block others being added retrospectively. With the system as-is, this will make no actual difference, but reduces the risk we miss this proc if introducing completion as a feature down the line.

Reply:
Makes sense, made the change so that apprenticeships are now matched if they have a Completed PaymentStatus.